### PR TITLE
Feature/supply kafka option

### PR DESCRIPTION
**Pull request by mikron**

### CODE REVIEW

File: infra/infra-queue/infra-queue-consumer/src/main/java/com/sflpro/notifier/queue/consumer/notification/push/KafkaConsumerListenerServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * Company: SFL LLC
+ * Created on 06/02/2018
+ *
+ * @author William Arustamyan
+ */
+@Service
+@ConditionalOnProperty(name = "queue.engine", havingValue = "kafka")
+public class KafkaConsumerListenerServiceImpl {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KafkaConsumerListenerServiceImpl.class);
```
Comment (sfl-sonar) [inline infra/infra-queue/infra-queue-consumer/src/main/java/com/sflpro/notifier/queue/consumer/notification/push/KafkaConsumerListenerServiceImpl.java]: MAJOR Remove this unused "LOGGER" private field. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)

File: core/core-services-impl/src/main/java/com/sflpro/notifier/services/notification/impl/push/sns/SnsArnConfigurationServiceImpl.java [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.config.PropertiesFactoryBean;
```
Comment (sfl-sonar) [inline core/core-services-impl/src/main/java/com/sflpro/notifier/services/notification/impl/push/sns/SnsArnConfigurationServiceImpl.java]: MINOR Remove this unused import 'org.springframework.beans.factory.config.PropertiesFactoryBean'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by sfl-sonar** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: core/core-db-repositories/src/main/resources/com/sflpro/notifier/repositories.properties [hunk 3]
```diff
@@ -10,7 +10,7 @@ database.password=
 
 flyway.locations=classpath:db/migration/{vendor}
 
-database.pool.maxActive=15
+database.pool.maxActive=1
```
Comment (dnavre) [inline core/core-db-repositories/src/main/resources/com/sflpro/notifier/repositories.properties]: ? :)

File: infra/infra-queue/infra-queue-producer/pom.xml [hunk 4]
```diff
@@ -43,6 +43,11 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka</artifactId>
+        </dependency>
```
Comment (dnavre) [inline infra/infra-queue/infra-queue-producer/pom.xml]: Move this dependency to queue-common

File: infra/infra-queue/infra-queue-consumer/pom.xml [hunk 5]
```diff
@@ -48,6 +48,10 @@
             <!--TODO chk with Mr. Smith-->
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka</artifactId>
+        </dependency>
```
Comment (dnavre) [inline infra/infra-queue/infra-queue-consumer/pom.xml]: Move this dependency to queue-common

**Pull request merged**